test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -130,6 +155,14 @@ describe('POST /group/:groupId/join', () => {
         await expect(invitedUser.get('/user')).to.eventually.not.have.deep.property('invitations.party.id');
       });
 
+      it('increments memberCount when joining party', async () => {
+        let oldMemberCount = party.memberCount;
```

Comment:
We are keeping the membership information purely in `user.party._id` if I'm not mistaken? Shouldn't `'allows invited user to join party'` test be enough?